Sources/Rendering/SceneGraph/ViewNode/index.js

```diff
@@ -184,6 +185,12 @@ function vtkViewNode(publicAPI, model) {
     }
     return ret;
   };
+
+  const parentDelete = publicAPI.delete;
+  publicAPI.delete = () => {
+    model.children.forEach((child) => child.delete());
```

Comment:
for performance (and GC) reasons, I would suggest to use a for loop instead.

Reply:
Switched to using a for loop. I figured delete wouldn't be called frequently enough for it to matter, but might as well not have to find out and fix it later.